wire-ios/Wire-iOS/Sources/UserInterface/UserProfile/ProfileViewController.swift

```diff
@@ -179,8 +177,8 @@ final class ProfileViewController: UIViewController {
         updateShowVerifiedShield()
     }
 
-    override func viewDidAppear(_ animated: Bool) {
-        super.viewDidAppear(animated)
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
```

Comment:
> Last but not least we replace viewDidAppear with viewWillApear so we can be the delay in which the x icon is shown when we present the view

I don't understand what you mean here, could you elaborate?

Reply:
Yes, so before that change there was a delay between presenting the view and showing the navigation bar buttons (take a look at the after video) But after making that change, there's no delay and the icons show when they should in the first place.

Reply:
@johnxnguyen Any more questions on this?

Reply:
Is it fine to setup the navigation items repeatedly?

Reply:
I think that's a different discussion and indeed I don't like that approach. I kid you not, I was just thinking that we need a plan to improve how we handle navigation, navigations controllers and in extent items. I think we have a brief discussion on this during the accessibility sprint. 

Happy to further discuss it and come up with a plan.